src/main/java/de/tum/in/www1/artemis/domain/iris/message/IrisMessageContent.java

```diff
@@ -0,0 +1,41 @@
+package de.tum.in.www1.artemis.domain.iris.message;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import de.tum.in.www1.artemis.domain.DomainObject;
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+import javax.annotation.Nullable;
+import javax.persistence.*;
+
+@Entity
+@Table(name = "iris_message_content")
+@Inheritance(strategy = InheritanceType.JOINED)
```

Comment:
I'm pretty sure you'd need to have a discriminator column now.

Reply:
Not strictly necessary but can't hurt